src/tests/ftest/server/metadata.py

```diff
@@ -330,7 +330,9 @@ def test_metadata_addremove(self):
                 # but make sure the number does not decrease more than 3 times
                 # in a row.
                 variation = quantity - containers_created[loop - 1]
-                if variation < 0:
+                # Variation is negative value and absolute variation greater
+                # than 5.
+                if variation < 0 and abs(variation) > 5:
```

Comment:
Couldn't we just
```suggestion
                # Variation is negative value and absolute variation greater
                # than 5.
                if variation < -5:
```

Reply:
@daltonbohning : Yes, we can... But, it looks like we need to change the verification logic based on the comments on DAOS-9078. It looks like the present logic of checking 3 consecutive variations may not be the correct method... I am checking with Ken to find the suitable method to implement here.